backend/compact-connect/lambdas/cloudfront-csp/test/index.test.js

```diff
@@ -209,27 +209,6 @@ describe(testFilename(__filename), () => {
 
             checkLambdaResult(environment, result);
         });
-        it('should successfully return the security headers for ia prod', async () => {
```

Comment:
why remove these tests?

Reply:
This test is for an environment that doesn't exist, so I removed it.